scss/components/_ckeditor.scss

```diff
@@ -0,0 +1,9 @@
+.ckeditor-toolbar-configuration {
+  .ckeditor-button {
+    a {
+      box-sizing: content-box;
+      -webkit-box-sizing: content-box;
```

Comment:
It is unnecessary to declare the prefixed attributes here. The build process will take care of it for us.